src/components/ebay-fake-menu/README.md

```diff
@@ -45,15 +45,16 @@
-| `value` (radio or checkbox)   | String  | No       | No                                     | the value to use with event responses for for the `checked` array           |
-| `checked` (radio or checkbox) | Boolean | No       | No                                     | whether or not the item is checked                                          |
-| `current` (fake menu)         | Boolean | No       | No                                     | whether or not the href is the current href of the page                     |
-| `badge-number`                | Number  | No       | No                                     | used as the number to be placed in the badge                                |
-| `badge-aria-label`            | String  | No       | Yes (only if badge number is provided) | passed as the `aria-label` directly to the badge                            |
+| Name                          | Type    | Stateful | Required                               | Description                                                                                                                                                          |
+| ----------------------------- | ------- | -------- | -------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `href` (fake menu)            | String  | No       | No                                     | for link that looks like a menu-item. If set to null then will disable item                                                                                          |
+| `type` (fake menu)            | String  | No       | No                                     | Set to "button" for fake menu-item `<button>`                                                                                                                        |
+| `value` (radio or checkbox)   | String  | No       | No                                     | the value to use with event responses for for the `checked` array                                                                                                    |
+| `checked` (radio or checkbox) | Boolean | No       | No                                     | whether or not the item is checked                                                                                                                                   |
+| `current` (fake menu)         | Boolean | No       | No                                     | whether or not the href is the current href of the page                                                                                                              |
+| `itemMatchesUrl` (fake menu)  | Boolean | No       | No                                     | used in conjunction with `current` -- This determines whether aria-current will be `page` or `true` -- Defaults to `true` which gives aria-current a value of `page` |
```

Comment:
I'm not sure I really like this name and how it works. It seems a bit off having to name it as `itemMatchesUrl=false`

Reply:
We are trying to abstract away the aria internals here, so that developers don't need knowledge of what is the correct ARIA attribute to use.

Reply:
We could also just go with `currentPage` or `currentItem` (and deprecate `current`). Might be cleaner.

Reply:
Also, for tabs I know we were trying to do this in a non-breaking way. Same applies here. For next major version we could look into making `current` an enumerated type.

Reply:
EDIT: actually, scrap that, I like Andrews approach better:

```
We can also make it so that anything other than true does not get set, so like:
$ var defaultAriaCurrent = item.a11yCurrentValue === 'true' ? 'true' : 'page';
```